lib/src/lifecycle_module.dart

```diff
@@ -569,6 +569,16 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
     return completer.future;
   }
 
+  /// This method is a workaround for the dangers of mock modules. A mock cannot implement a private member, so this will fail.
+  void _setParentContextOnChild(
+    LifecycleModule? childModule,
+    SpanContext? context,
+  ) {
+    try {
+      childModule?._parentContext = context;
+    } catch (e) {}
```

Comment:
It might be good to add a comment inside the `{}` saying that swallowing the exception and not logging or acting on it is intentional.